### DISCUSSION TITLE
How long does github support take

### DISCUSSION THREAD

**Discussion by RyanLangston** [community/community#47459] [Codespaces] [answered]
Hey, so I've been having a billing issue related to Codespace prebuild (paying for something that no longer even exists) and have been waiting for 8 days as of the time of writing. I was wondering how long this kind of issue usually takes to resolve or receive a response.

**Comment by jge162** [comment, marked as answer]
It will depend I think on the queue size. FIFO.

**Reply by queenofcorgis** [reply]
@jge162 is correct - it depends on both how many folks are ahead of you and our team's resources. We appreciate your patience!

**Reply by RyanLangston** [reply]
Alright, I guess I'll have to grow some patience, thanks for the response :)

**Reply by queenofcorgis** [reply]
You're welcome - I did take a look and can confirm your ticket is in the right place so hang tight and our trusty Supportocats will be in touch.
